Libraries/LibJS/Runtime/Date.h

```diff
@@ -40,6 +40,9 @@ class Date final : public Object {
     String date_string() { return m_datetime.to_string("%a %b %d %Y"); }
     // FIXME: Deal with timezones once SerenityOS has a working tzset(3)
     String time_string() { return m_datetime.to_string("%T GMT+0000 (UTC)"); }
+    String string() {
```

Comment:
`{` placement :)
Also, this function could be const-qualified (together with `date_string()` and `time_string()`)